storage-proofs/post/Cargo.toml

```diff
@@ -38,5 +39,5 @@ filecoin-hashers = { path = "../../filecoin-hashers", version = "1.0.0", default
 [features]
 default = ["pairing", "gpu"]
 gpu = ["storage-proofs-core/gpu", "filecoin-hashers/gpu"]
```

Comment:
this and the other crates depending on `fr32` will now need to enable the `gpu` feature